### DISCUSSION TITLE
Azuracast covers disappear on reboot

### DISCUSSION THREAD

**Discussion by Artem89233** [AzuraCast/AzuraCast#7547] [Q&A]
I am using docker installation on Proxmox VM. 
And I ran into a problem, Azuracast does not save the covers of the song even after restarting the computer.
My song catalog is mounted using NFS.

After the reboot, you have to re-process the music files:
Before the reboot:
![изображение](https://github.com/user-attachments/assets/2b3067b4-caf3-4e49-b7e8-2455524a8663)
After the reboot:
![изображение](https://github.com/user-attachments/assets/186a5bee-0b46-4543-a9dd-e465a342fdfe)
And after the reboot, I processed and the cover appeared:
![изображение](https://github.com/user-attachments/assets/f7f98a95-8a49-4304-885d-ce6d30241dc0)

What could be the problem?

**Comment by BusterNeece** [comment]
@Artem89233 Sounds like you don't have persistent Docker volumes set up on your installation.

You should mount either a filesystem or a persistent volume to the following directories:
 - `/var/azuracast/stations` (for station media and data)
 - `/var/lib/mysql` (for the MariaDB database)
 - `/var/azuracast/storage` (for uploads and branding)
 - `/var/azuracast/backups` (to store automated backups)

**Comment by gualtierogrey** [comment]
Having same problem, same proxmox method but uploaded via browser. Covers never shown up so just default cover. Installed backup from working Ubuntu server without Media so must not be related of way of mounting
